ow_plexil/src/plexil-adapter/OwInterface.h

```diff
@@ -252,9 +254,9 @@ class OwInterface : public PlexilInterface
   };
 
   FaultMap64 m_panTiltErrors = {
-    {"PAN_JOINT_LOCKED", std::make_pair(
+    {"PanFault", std::make_pair(
       owl_msgs::PanTiltFaultsStatus::PAN_JOINT_LOCKED, false)},
-    {"TILT_JOINT_LOCKED", std::make_pair(
+    {"TiltFault", std::make_pair(
```

Comment:
```suggestion
    {"TILT_JOINT_LOCKED", std::make_pair(
```
This reverts a telemetry unification update, probably not intended?